tools/CaptureLogs/iot_providers.txt

```diff
@@ -0,0 +1,5 @@
+"{e927240b-7198-5cc8-72f1-7ddcf31bb8cb}"
```

Comment:
Add readme to capture guid to event source mapping and delete the guid from source code comments

Reply:
👍 We need to point to https://github.com/dotnet/corefx/blob/master/Documentation/debugging/windows-instructions.md#traces and also document our own mapping.
Adding a quick readme.md but we'll need to enhance it later.

Reply:
Done: added readme.md in the tools folder.